### PR TITLE
Fix appinsights

### PR DESCRIPTION
**Pull request by timja**
Correct app insights integration

### CODE REVIEW

File: lib/AI-Agent.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="utf-8"?>
+<ApplicationInsightsAgent>
+    <AgentLogger>INFO</AgentLogger>
+    <Instrumentation>
+        <BuiltIn enabled="true">
+            <HTTP enabled="true"/>
+        </BuiltIn>
+        <Class name="uk.gov.hmcts.cmc.claimstore.repositories.DBCaseRepository">
```
Comment (tzarsmango) [inline lib/AI-Agent.xml]: not a ccd class

File: docker-compose.yml [hunk 2]
```diff
@@ -40,8 +40,8 @@ services:
       timeout: 10s
       retries: 10
     environment:
-      - DATA_STORE_DB_USERNAME
-      - DATA_STORE_DB_PASSWORD
+      - DATA_STORE_DB_USERNAME=data_store
```
Comment (tzarsmango) [inline docker-compose.yml]: why user/pass change

File: docker-compose.yml [hunk 3]
```diff
@@ -10,8 +10,8 @@ services:
     environment:
       - DATA_STORE_DB_HOST=ccd-data-store-database
       - DATA_STORE_DB_PORT=5432
-      - DATA_STORE_DB_USERNAME
-      - DATA_STORE_DB_PASSWORD
+      - DATA_STORE_DB_USERNAME=data_store
```
Comment (tzarsmango) [inline docker-compose.yml]: why user/pass change

File: lib/AI-Agent.xml (hunk 1 above)
Comment (timja) [reply lib/AI-Agent.xml]: mario is working on this, its a draft PR

File: docker-compose.yml (hunk 2 above)
Comment (timja) [reply docker-compose.yml]: to make it usable locally without needing to do anything

**Pull request closed without merging**